build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/ArtifactoryClientConfiguration.java

```diff
@@ -529,6 +528,52 @@ public void setUseDotnetCli(boolean useDotnetCli) {
         }
     }
 
+    public class DockerHandler extends PrefixPropertyHandler {
+        public DockerHandler() {
+            super(root, PROP_DOCKER_PREFIX);
+        }
+
+        public String getImageTag() {
+            return rootConfig.getStringValue(DOCKER_IMAGE_TAG);
+        }
+
+        public void setImageTag(String imageTag) {
+            rootConfig.setStringValue(DOCKER_IMAGE_TAG, imageTag);
+        }
+
+        public String getHost() {
+            return rootConfig.getStringValue(DOCKER_HOST);
+        }
+
+        public void setHost(String host) {
+            rootConfig.setStringValue(DOCKER_HOST, host);
+        }
+
+        public Map<String, String> getArtifactProperties() {
```

Comment:
The logic of extracting the props from a duplicationstring is duplicated inside the build-info code base. Let's consolidate and reuse it.
The same goes for the below setArtifactProperties method. We should avoid logic duplication.